filament/src/details/MaterialInstance.h

```diff
@@ -156,6 +158,7 @@ class FMaterialInstance : public MaterialInstance {
     backend::RasterState::DepthFunc mDepthFunc;
 
     uint64_t mMaterialSortingKey = 0;
+    utils::CString mName;
```

Comment:
I would put this last (because it's rarely used) and also make it `const`.

Reply:
I didn't make it const because it causes C++ to complain about the default constructor, which might be why Material's name isn't const either.

Reply:
Moved it to the end.